DeqingSun/USB_flashlight_Glass
# Add Gradle build files

Allows project to be built using Android Studio. Updates .gitignore to ignore unnecessary generated project files.

Test Plan:
1) Download the latest Android Studio (0.5.8)
2) Select import project and use Gradle import. Ensure that project imports and builds successfully
